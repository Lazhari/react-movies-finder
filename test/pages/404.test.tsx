import React from 'react'
import { render } from '../testUtils'
import NotFound from '../../pages/404'

describe('404 page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<NotFound />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(<Home />, {})
  //   window.alert = jest.fn()
  //   // fireEvent.click(getByText('Test Button'))
  //   // expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
  // })
})
