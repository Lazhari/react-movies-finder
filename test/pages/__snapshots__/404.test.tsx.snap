// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root makeStyles-root-1 MuiContainer-maxWidthXl"
  >
    <div>
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root"
        focusable="false"
        style="font-size: 160px;"
        viewBox="0 0, 60 60"
      >
        <path
          d="M57 0H3C1.346 0 0 1.346 0 3v56a1 1 0 001 1h58a1 1 0 001-1V3c0-1.654-1.346-3-3-3zM3 2h54c.551 0 1 .449 1 1v5H2V3c0-.551.449-1 1-1zm16 51v5h-8v-5h8zm2 0h8v5h-8v-5zm10 0h8v5h-8v-5zm10 0h8v5h-8v-5zM2 51V17h56v34H2zm0-36v-5h56v5H2zm0 38h7v5H2v-5zm49 5v-5h7v5h-7z"
        />
        <path
          d="M5 4h4v2H5zM12 4h4v2h-4zM16 24h-2v13H8V24H6v14a1 1 0 001 1h7v6h2v-6h3v-2h-3V24zM54 37h-3V24h-2v13h-6V24h-2v14a1 1 0 001 1h7v6h2v-6h3v-2zM30 24c-3.86 0-7 3.14-7 7v7c0 3.86 3.14 7 7 7s7-3.14 7-7v-7c0-3.86-3.14-7-7-7zm5 14c0 2.757-2.243 5-5 5s-5-2.243-5-5v-7c0-2.757 2.243-5 5-5s5 2.243 5 5v7z"
        />
      </svg>
    </div>
    <h1
      class="MuiTypography-root makeStyles-title-2 MuiTypography-h5 MuiTypography-gutterBottom MuiTypography-alignCenter"
    >
      Page Not Found
    </h1>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
    >
      Oops! We can't find the page you are looking for!!
    </p>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-homeButton-3"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Back to Home
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
